delira/training/callbacks/abstract_callback.py

```diff
@@ -59,3 +59,41 @@ def at_epoch_end(self, trainer, **kwargs):
 
         """
         return {}
+
+    def at_iter_begin(self, trainer, **kwargs):
+        """
+        Function which will be executed at begin of each iteration
+
+        Parameters
+        ----------
+        trainer : :class:`AbstractNetworkTrainer`
+        **kwargs :
+            additional keyword arguments
+
+        Returns
+        -------
+        dict
+            modified trainer attributes, where the name must correspond to the
+            trainer's attribute name
+
+        """
+        return {}
+
+    def at_iter_end(self, trainer, **kwargs):
```

Comment:
Explicitly define which arguments are passed here.

Reply:
Same as above

Reply:
I'm closing this because it is the same as `at_iter_begin` .